packages/core/src/tracing/clsHooked/async_local_storage_context.js

```diff
@@ -20,6 +20,8 @@ const unset = require('./unset');
 
 const CONTEXTS_SYMBOL = 'instanaClsHooked@contexts';
 
+const DEBUG_CLS = !!process.env.DEBUG_CLS;
```

Comment:
Good point. I wonder if we even want to keep this debug option at all. It was useful for testing and troubleshooting, but the two log statements are a bit arbitrary I think. How do you feel about removing this completely?

Reply:
Good question. I was wondering about this as well, but having debug logs available in case there is a problem with a customer is quite nice. 

I would say: up to you 👍 Either remove, leave as is or optimise the usage of the env variable :D

Reply:
I'll remove it for now.